spec/customer_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'customers'
+
+describe Customers do
+
+    let(:my_customer) { Customers.new }
+
+    describe "#create_customer" do
+        it "Should add a new customer with a name and number" do
+            mobile_number = 7999985111
+            expect(my_customer.create_customer("Joe Bloggs", mobile_number)).to eq mobile_number
```

Comment:
Why does creating a new customer only return the mobile number? Could it return a hash where name: "Joe Bloggs" and mobile_number: = 7999985111